.gitignore

```diff
@@ -3,4 +3,5 @@
 Gemfile.lock
 pkg/*
 *.DS_Store
-.rvmrc
\ No newline at end of file
+spec/db/test.db
+.ruby-version
```

Comment:
yeah, for a gem, it should work with a variety of versions, I don't want to require a specific version. Thoughts?